script.js

```diff
@@ -299,3 +310,45 @@ workMobile.forEach((work) => {
   });
   mobileWork.appendChild(item);
 });
+
+// form validatio
+
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224444645-aae6b8f4-9e00-4678-a333-ab50a03e41b0.png)

**[Optional]**

- [ ] Kindly remove this NOTE to make your readme file descriptive and clean 😊 This NOTE is for you when you create your license file and I can see you already have it so there is no point of having this NOTE in your readme 🤔